index.html

```diff
@@ -47,6 +47,183 @@ <h1>THIERRY <br> NOUMBUH</h1>
+                    <div>
+                        <ul class="project-languages">
+                            <li class="language-css">HTML/CSS</li>
+                            <li class="language-css">Ruby on Rails</li>
+                            <li class="language-css">JavaScript</li>
+                        </ul>
+                    </div>
+                    <button class="project-button">
+                    <span>See this project</span> 
+                        <img src="assets/images/arrow.svg" alt="arrow" class="arrow">
+                    </button>
+                </div>
+            </section>
+
+                <section class="containing-6">
+                    <div class="rectangle_21">
+                        <img src="assets/images/rectangle_23.svg" alt="rectangle_21" />
+                    </div>
+                    <div class="rectangle_22">
+                        <span class="project-name">project name goes here</span>
+                        <div>
+                            <ul class="project-languages">
+                                <li class="language-css">HTML/CSS</li>
+                                <li class="language-css">Ruby on Rails</li>
+                                <li class="language-css">JavaScript</li>
+                            </ul>
+                        </div>
+                        <button class="project-button">
+                        <span>See this project</span> 
+                            <img src="assets/images/arrow.svg" alt="arrow" class="arrow">
+                        </button>
+                    </div>
+                </section>
+
+                    <section class="containing-6">
+                        <div class="rectangle_21">
+                            <img src="assets/images/rectangle_24.svg" alt="rectangle_21" />
+                        </div>
+                        <div class="rectangle_22">
+                            <span class="project-name">project name goes here</span>
+                            <div>
+                                <ul class="project-languages">
+                                    <li class="language-css">HTML/CSS</li>
+                                    <li class="language-css">Ruby on Rails</li>
+                                    <li class="language-css">JavaScript</li>
+                                </ul>
+                            </div>
+                            <button class="project-button">
+                            <span>See this project</span> 
+                                <img src="assets/images/arrow.svg" alt="arrow" class="arrow">
+                            </button>
+                        </div>
+                    </section>
+
+                        <section class="containing-6">
+                            <div class="rectangle_21">
+                                <img src="assets/images/rectangle_25.svg" alt="rectangle_21" />
```

Comment:
- [ ] Also, please, make sure your projects images don't have `margin` or `padding` to stick to the design as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍 

 ### Exemple

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-06-01 161622](https://github.com/axcell-0/Thierrys-portfolio/assets/57408419/0e82b0c8-9a75-4d89-ba9a-188b2590bd5f)| |![Screenshot 2023-06-01 161600](https://github.com/axcell-0/Thierrys-portfolio/assets/57408419/4f9f3e7b-50de-4ce6-af0f-b59bd7801579)| |


> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-06-01 161750](https://github.com/axcell-0/Thierrys-portfolio/assets/57408419/20c8a0c9-37bd-418d-9bf9-cd935ea00f9c)| |![Screenshot 2023-06-01 161718](https://github.com/axcell-0/Thierrys-portfolio/assets/57408419/1fe20b86-d76e-4b18-b3cb-065dc7305218)| |